### ISSUE TITLE
Basic UI: visibility attribute not dynamically taken into account

### ISSUE THREAD

**Issue by lolodomo** [eclipse/smarthome#1138]
Visibility attribute seems to be correctly taken into account by Basic UI when opening a page.
But if items are updated triggering visibility changes, the changes of items visibility are not applied in the page except if you re-open it.

**Comment by lolodomo** [comment]
This issue is apparently a known issue already declared: #640

**Comment by lolodomo** [comment]
@resetnow @kaikreuzer : I just tested again this issue (I am running snapshot 554) and I can see now that items are displayed / hidden automatically without reloading the frame. The problem I noticed is that certain items are displayed with a non updated value. I can see that with the Sonos binding and "artist" and "album" items. I have one frame that uses visibility attribute for few items including these ones and another frame that displays all the items of my Sonos group, that is without the use of the visibility attribute. In this second frame, I can see that the same items are updated as expected without doing anything. So my conclusion is that there is still something wrong with item updates in Basic UI only in the case you use visibility attribute.
Hoping I was clear in my explanations.
